scripts/sysman/driver.yml

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Tool APIs for System Resource Management (Sysman)"
+ordinal: "1"
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Supported sysman initialization flags"
+version: "1.5"
+class: $s
+name: $s_init_flags_t
+etors:
+    - name: PLACEHOLDER
+      desc: "placeholder for future use"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Initialize $OneApi System Resource Management (sysman)"
+version: "1.5"
+class: $s
+name: Init
+decl: static
+ordinal: "0"
+details: 
+    - "The application must call this function or $xInit with the $S_ENABLE_SYSMAN environment variable set before calling any other sysman function."
```

Comment:
@bashbaug : I think we should clarify that zeInit is still needed or not. what do you think? something like:

"Applications wishing to use Core APIs alongside sysman functions should call zeInit as well".

I dont think we have restrictions on which should be called first, whether zeInit or zesInit.

Reply:
@jitendrasharma1989 : Agree. That's why we need to clarify that.

Reply:
I think the specification is correct as-written:

* Applications must call zesInit (or zeInit with ZES_ENABLE_SYSMAN) before calling any sysman functions.  This requirement is documented here.
* Applications must call zeInit before calling any core API functions.  This requirement is documented as part of zeInit.
* There is no ordering requirement.

I don't mind adding another line cautioning that this function does NOT initialize the core APIs, just to remove possible confusion, but I do think it is extraneous.

Regarding why we still support initializing sysman with zeInit + ZES_ENABLE_SYSMAN, this is because removing this functionality would break backwards compatibility so we cannot do it in a minor API revision.  This is something we should do when we make a major API revision, though.

Reply:
thanks @bashbaug . I would say then to just add the line about the ordering requirement. Wording about zeInit vs zesInit calls is good then as you say.

Reply:
I added two additional lines for clarification:

> * This function will only initialize sysman. To initialize other functions, call zeInit.

> * There is no requirement to call this function before or after zeInit.